# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# dudumomo <dudumomo@gmail.com>, 2014
# Julien Syx <nephilim.seraphin@gmail.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: LISA\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-08-15 08:50+0200\n"
"PO-Revision-Date: 2014-08-15 06:55+0000\n"
"Last-Translator: Julien Syx <nephilim.seraphin@gmail.com>\n"
"Language-Team: French (http://www.transifex.com/projects/p/lisa/language/fr/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: fr\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: ../libs/commands.py:42
#, python-format
msgid "The client %(from)s has joined the zone %(zone)s"
msgstr "Le client %(from)s a rejoint la zone %(zone)s"

#: ../libs/rulesengine.py:35
#, python-format
msgid "Before 'before' rule: %(jsonInput)s"
msgstr "Avant la règle 'before' : %(jsonInput)s"

#: ../libs/rulesengine.py:39
#, python-format
msgid "After 'before' rule: %(jsonInput)s"
msgstr "Après la règle 'before' : %(jsonInput)s"

#: ../libs/rulesengine.py:52
msgid "I have not the right plugin installed to answer you correctly"
msgstr "Je n'ai pas le module adéquat pour répondre à votre demande"

#: ../libs/rulesengine.py:55
#, python-format
msgid "Before 'after' rule: %(jsonOutput)s"
msgstr "Avant la règle 'after' : %(jsonOutput)s"

#: ../libs/rulesengine.py:62
#, python-format
msgid "After 'after' rule: %(jsonOutput)s"
msgstr "Après la règle 'after' : %(jsonOutput)s"

#: ../libs/server.py:73 ../libs/websocket.py:88
#, python-format
msgid "Lost connection.  Reason: %(reason)s"
msgstr "Connexion perdue. Raison : %(reason)s"

#: ../libs/server.py:84
msgid "Error : Invalid JSON"
msgstr "Erreur : JSON invalide"

#: ../libs/server.py:105
msgid "Reloading L.I.S.A Engine"
msgstr "Rechargement du moteur L.I.S.A"

#: ../libs/server.py:110
msgid "Reloading Task Scheduler"
msgstr "Rechagement du plannificateur de tâches"

#: ../libs/websocket.py:78
msgid "Started to connect."
msgstr "Connexion en cours."

#: ../libs/websocket.py:82
msgid "Connected to Lisa."
msgstr "Connexion établie."

#: ../libs/websocket.py:83
msgid "Resetting reconnection delay"
msgstr "Paramétrage du délai de reconnexion"

#: ../libs/websocket.py:92
#, python-format
msgid "Connection failed. Reason: %(reason)s"
msgstr "Connexion échouée. Raison: %(reason)s"

#: ../plugins/PluginManager.py:57
msgid "Plugin already installed"
msgstr "Plugin déjà installé"

#: ../plugins/PluginManager.py:129
msgid "Plugin installed"
msgstr "Plugin installé"

#: ../plugins/PluginManager.py:138
msgid "Plugin already enabled"
msgstr "Plugin déjà activé"

#: ../plugins/PluginManager.py:153
msgid "Plugin enabled"
msgstr "Plugin activé"

#: ../plugins/PluginManager.py:162
msgid "Plugin already disabled"
msgstr "Plugin déjà désactivé"

#: ../plugins/PluginManager.py:178
msgid "Plugin disabled"
msgstr "Plugin désactivé"

#: ../plugins/PluginManager.py:186 ../plugins/PluginManager.py:316
msgid "Plugin not installed"
msgstr "Plugin pas installé"

#: ../plugins/PluginManager.py:200
msgid "Plugin uninstalled"
msgstr "Plugin désinstallé"

#: ../plugins/PluginManager.py:238
msgid "Plugin already exist on Pypi"
msgstr "Le plugin existe déjà sur Pypi"

#: ../plugins/PluginManager.py:244
msgid "Plugin already exist in the store"
msgstr "Le plugin existe déjà dans le store"

#: ../plugins/PluginManager.py:307
msgid "Plugin created"
msgstr "Plugin créé"

#: ../plugins/PluginManager.py:331
msgid "The json of the plugin can't be loaded"
msgstr "Le json contenu dans le plugin ne peut être chargé"

#: ../plugins/PluginManager.py:394
msgid "Plugin already up to date"
msgstr "Plugin déjà à jour"

#: ../plugins/PluginManager.py:395
msgid "Plugin upgraded"
msgstr "Plugin mis à jour"

#: ../plugins/PluginManager.py:418
msgid "Singleton can't be created twice !"
msgstr "Le Singleton ne peut être créé deux fois"
